styles.css

```diff
@@ -0,0 +1,104 @@
+* {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+  text-decoration: none;
+  box-sizing: border-box;
+}
+
+body {
+  background-image: url(./images/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  background-position: 100%;
+  background-size: contain;
+}
+
+html {
+  font-family: 'poppins', sans-serif;
+  font-size: 16px;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 2rem;
+}
+
+.logo a {
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+.humberger i {
+  color: #6070ff;
+  font-size: 18px;
+  line-height: 20px;
+  font-weight: 700;
+}
+
+section {
+  padding: 2rem;
+  max-width: 378px;
+}
```

Comment:
- [ ] Kindly make sure to align the headline intro contents by using CSS `flexbox` which gives us a lot of options to align all our elements, plus, it's responsive. Please make sure to provide enough space between the contents in order to match the provided Figma design.

  - ### YOUR DESIGN
  
     ![image](https://user-images.githubusercontent.com/87197412/235483756-640438d1-903d-4d32-aa44-1944fefb2344.png)

  - ### FIGMA DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/235483839-61d7cabc-11d7-452d-98a4-c615cdcb2230.png)